go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/eraser
 
-go 1.16
+go 1.18
```

Comment:
we don't need to add this as we don't rely on go 1.18 features yet

Reply:
Reverted this back to `1.16`